src/common/services/personApi.js

```diff
@@ -98,15 +98,17 @@ module.exports = {
    *
    * @param {String} id id of entity performing action
    * @param {String} userType type of user performing action in ['individual', 'organisation']
+   * @param {Number} page page of interested
    * @returns {Promise} resolves with API response
    */
-  getPeople(id, userType) {
+  getPeople(id, userType) { // , page) {
```

Comment:
Remove this commented out code

Reply:
lol, hangover from the attempt at people screen pagination. Will remove.